packages/api/__tests__/api/public/block.graphql

```diff
@@ -72,8 +72,17 @@ fragment FullBlock on Block {
 
   ... on LinkPageBreakBlock {
     text
+    richText
```

Comment:
You don't need to update the tests if you don't intend to test the changes.

Reply:
does it harm?

Reply:
if it doesn't do anything don't add it.

Reply:
i d suggest to merge and then directky add test with your team to understand the extended block better

Reply:
just remove it from the PR